components/scream/tests/uncoupled/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ if (NOT SCREAM_BASELINES_ONLY)
   else()
     message(STATUS "RRTMGP only supported for double precision builds; skipping")
   endif()
-  if (SCREAM_ENABLE_EXPERIMENTAL)
+  if (SCREAM_TEST_LEVEL GREATER_EQUAL SCREAM_TEST_LEVEL_EXPERIMENTAL)
```

Comment:
Have we checked if the code will build at the experimental level with this flag.  I think the ZM sub-directory is so old and unsupported that it may not even build now.  Maybe we can comment this out and add a comment that ZM is no longer supported even experimentally.

Reply:
I did not check that and I'm not worried about it in this PR unless you think it is urgent to remove ZM. The new code is functionally identical to the old, so ZM problems will not be created by this PR.